### DISCUSSION TITLE
[UI issue] blue box appears when I click a line

### DISCUSSION THREAD

**Discussion by tanhenggek** [ohmyzsh/ohmyzsh#12446] [Help] [answered]
In the latest update, I realize a blue box appears whenever I click a line. Please refer to the attached screenshot. It's self-explanatory. Please share your solution if you know to turn off this feature. Thank you.

<img width="1317" alt="000786" src="https://github.com/ohmyzsh/ohmyzsh/assets/8288307/902b9857-41b5-400c-b251-3cd6529b07b9">

**Comment by carlosala** [comment]
That is not related to `oh-my-zsh`. Looks like it might be related to [iTerm2 Shell Integration](https://iterm2.com/documentation-shell-integration.html)

**Comment by tanhenggek** [comment]
@carlosala you're right. iTerm2 latest release includes this UI feature. Thanks.

https://iterm2.com/downloads/stable/iTerm2-3_5_0.changelog

**Comment by tanhenggek** [comment, marked as answer]
for those who came across this issue and thread, you may find the solution here. https://gitlab.com/gnachman/iterm2/-/issues/11532
